index.html

```diff
@@ -4,26 +4,10 @@
     <link href="https://fonts.googleapis.com/css2?family=Stalinist+One&display=swap" rel="stylesheet"> 
     <title>Durak Project</title>
   </head>
-  <body>
+  <body>   
```

Comment:
wtf fam stop the trailing spaces

Reply:
don't worry about this, I'm going to set up auto-generation for this index file anyway so it's pretty much irrelevant